src/ir/resources/mod.rs

```diff
@@ -473,6 +455,30 @@ impl ResourceType {
                 }
                 Ok(Self::Custom(name.into()))
             }
+            "Alexa" => {
+                let service = match parts.next() {
+                    Some("") | None => {
+                        return Err(TransmuteError::new(format!(
+                            "invalid resource type: {from:?} (missing service name)"
+                        )))
+                    }
+                    Some(service) => service.into(),
+                };
+                let type_name = match parts.next() {
+                    Some("") | None => {
+                        return Err(TransmuteError::new(format!(
+                            "invalid resource type: {from:?} (missing resource type name)"
+                        )))
+                    }
+                    Some(type_name) => type_name.into(),
+                };
+                if parts.next().is_some() {
```

Comment:
Will this fail if someone passes a hook or module? I'm not sure exactly how thes einteract but I think they have 4 segments and the last is like ::hook